example/osp_provision_example.py

```diff
@@ -98,10 +98,13 @@
 
 vm_name = payload_data["vm_name"]
 # 7. Report the floating ip address back to the user
+# need an extra sleep to make sure floating ip is added to
+# the instance after provisioning
+sleep(10)
```

Comment:
Just a comment: I think it is okay for this example script to have the sleep to ensure the floating IP is attached to the instance. In the future when a script is provided to handle all of this, this area should be enhanced that when doing the query if it is unable to get the floating IP, have it do some sort of retry. Maybe give X amount of retries before failing. I just am thinking of network latency or connectivity issues could cause 10 seconds maybe to not be enough time.